modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators.Tests/BaseTest.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.CodeAnalysis;
+
+namespace Godot.SourceGenerators.Tests;
+
+public abstract class BaseTest<TSourceGenerator>
```

Comment:
I was a bit confused by the name of this class. This isn't a test, it's a test suite.